semgrep-core/src/core/Semgrep_error_code.mli

```diff
@@ -39,12 +48,6 @@ val exn_to_error :
 val try_with_exn_to_error : Common.filename -> (unit -> unit) -> unit
 val try_with_print_exn_and_reraise : Common.filename -> (unit -> unit) -> unit
 
-(*
```

Comment:
so this is fixed? It does not take 2 seconds anymore?

Reply:
I didn't run into the 2-second delay on the example above on my machine, which was the problem. I think it was fixed when we upgraded OCaml a while ago (I forgot which version; maybe when we went from 4.10 to 4.12).